consult.el

```diff
@@ -451,7 +451,17 @@ Used by `consult-completion-in-region', `consult-yank' and `consult-history'.")
      :foreground "#333"))
   "Face used for thin line separators in `consult-register-window'.")
 
-;;;; History variables
+;;;; Command argument variables (public)
+
+(defvar consult-last-command-args nil
+  "List of args used in prior consult command.
+Automatically set in `consult--async-process'.")
+
+(defvar consult-this-command-args nil
+  "List of args used in current consult command.
+Automatically set in `consult--async-process'.")
```

Comment:
These global variables should not be necessary. Please remove them.

Reply:
You are right, this was a separate feature.  I thought it would be useful to be able to quickly manipulate the actual command-line arguments used as I found it hard to see this information when initially working with the code.  I patterned it after this-command and last-command in Emacs as it can serve a similar function.